watermarker/src/commonTest/kotlin/unitTest/fileWatermarker/TextWatermarkerTest.kt

```diff
@@ -206,6 +206,64 @@ class TextWatermarkerBuilderTest {
         assertNull(result.value)
     }
 
+    /**
+     * tests if the validate() function of the builder will correctly return an error when
+     * either Separator in the StartEnd Strategy is an illegal Char, as well as when both the
+     * Start and End Char are illegal
+     */
```

Comment:
Writing comments for functions is always good. But since we decided to not comment any tests, this comment should be deleted to have a uniform code.